src/export/export.go

```diff
@@ -75,6 +71,22 @@ func export(graph *core.BuildGraph, dir string, target *core.BuildTarget, done m
 			}
 		}
 	}
+}
+
+// export implements the logic of ToDir, but prevents repeating targets.
+func export(graph *core.BuildGraph, dir string, target *core.BuildTarget, done map[*core.BuildTarget]bool) {
+	if done[target] {
+		return
+	}
+	// We want to export the package that made this subrepo available, but we still need to walk the target deps
+	// as it may depend on other subrepos or first party targets
+	if target.Subrepo != nil {
+		export(graph, dir, target.Subrepo.Target, done)
+	}
+	if target.Subrepo == nil {
```

Comment:
```suggestion
	else {
```

(am I missing something, or is the GitHub UI for suggested edits terrible?)